doc/Caliptra.md

```diff
@@ -377,11 +377,11 @@ The owner key, when represented in fuses or in the FMC's alias certificate, is a
 5. SoC follows the boot flow as defined in Caliptra IP HW boot flow to assert cptra\_pwrgood and deassert cptra\_rst\_b, followed by writing to the fuse registers.
 6. HVM, through JTAG or using the Caliptra SoC interface, sets “CPTRA\_DBG\_MANUF\_SERVICE\_REG” bit 0 to request a CSR.
 7. HVM, through JTAG or using the Caliptra SoC interface, writes to “CPTRA\_BOOTFSM\_GO” to allow Caliptra’s internal BootFSM to continue to bring up microcontroller out of reset.
-8. ROM looks at the manufacturing state encoding, “CPTRA\_DBG\_MANUF\_SERVICE\_REG”, and populates the Caliptra internal SRAM \[the mailbox SRAM hardware structure is reused\] with the CSR.
+8. ROM reads the manufacturing state encoding from the register “CPTRA\_DBG\_MANUF\_SERVICE\_REG”, acquires the mailbox lock, and populates the Caliptra internal SRAM (the mailbox SRAM hardware structure is reused) with the CSR.
```

Comment:
recommend reordering slightly to: `...from the “CPTRA\_DBG\_MANUF\_SERVICE\_REG” register, acquires...`

Reply:
done!